packages/components/src/ItemList.test.tsx

```diff
@@ -17,8 +17,8 @@ function makeItemList({
   rowHeight = 20,
   offset = 0,
   items = makeItems(),
-  onSelect = () => {},
-  onViewportChange = () => {},
+  onSelect = () => false,
+  onViewportChange = () => false,
```

Comment:
`false`? Why that instead of `undefined`?

Reply:
No reason. Maybe I didn't feel like typing out undefined that day. I changed it to undefined now.